websites/L/LITT Live/presence.ts

```diff
@@ -9,12 +9,16 @@ const presence = new Presence({
 let songName: HTMLElement,
 	songArtist: HTMLElement,
 	songNameS: string,
-	songArtistS: string;
+	songArtistS: string,
+	thumbnail: HTMLImageElement;
 
 presence.on("UpdateData", async () => {
+	if (document.querySelectorAll(".MuiPaper-elevation")[0] && document.querySelectorAll(".MuiPaper-elevation")[0].childNodes[1]) 
+		thumbnail = document.querySelectorAll(".MuiPaper-elevation")[0].childNodes[1].childNodes[0].childNodes[0].childNodes[0] as HTMLImageElement;
```

Comment:
* `querySelectorAll(...)[0]` can be simplified to `querySelector(...)`.
* Considering the multiple uses of the same selector, assign the element to a variable.